deploy/eks/README.md

```diff
@@ -0,0 +1,67 @@
+# kiam eks cluster setup
+
+This is a simple example how i got kiam working with an eks cluster completly running on spot instances which assumes instance role instead of having them directly assigned. Please change the server / agent deployment to your needs. I added node taints and selectors because kiam needs a dedicated node which doesn't proxy requests to the server. I also like the idea with an dedicated instance to give less privileges to an node and have the setup more secure.
```

Comment:
why does KIAM *need* a dedicated node that doesn't proxy requests to the server?

Reply:
It's not necessary, but highly recommended. Placing servers on nodes that aren't available to cluster user workloads helps reduce the IAM permissions accessible to machines running user processes. Instead, only the node running the Kiam server needs to be able to assume all the roles in your cluster.